.gitignore

```diff
@@ -14,5 +14,5 @@ env/
 node_modules/
 out/
 parts/
-sql/
+sql
```

Comment:
this will ignore the file `sql` not the folder `sql/` is that what we want here ? @nslater